documentation/assemblies/metrics/assembly_metrics-kafka-bridge.adoc

```diff
@@ -0,0 +1,11 @@
+// This assembly is included in the following assemblies:
+//
+// metrics/assembly-metrics.adoc
+
+[id='assembly-kafka-bridge-{context}']
+= Add Kafka Bridge
```

Comment:
It is not clear from the PR what will be the context ... but the title `Add Kafka Bridge` sounds weird. Should it be just `Kafka Bridge`?

Reply:
Yes agree, I was using the Kafka Exporter section as reference but it doesn't make sense "Add" here.

Reply:
To keep with the verb/object style of the other sub-headings in the chapter, maybe we can use _Monitor Kafka Bridge instances_